static/js/components/ProfileImage.js

```diff
@@ -7,18 +7,39 @@ import type { Profile } from '../flow/profileTypes';
 
 export default class ProfileImage extends React.Component {
   props: {
-    profile: Profile
+    profile: Profile,
+    editable: boolean
+  };
+
+  static defaultProps = {
+    editable: false
   };
 
   render () {
-    const { profile } = this.props;
+    const { profile, editable } = this.props;
     const imageUrl = makeProfileImageUrl(profile);
+    let cameraIcon = () => {
+      if (editable) {
+        return (
+          <span className="img">
+            <img
+              src='/static/images/camera.png'
+              alt="Camera image for profile image"
+            />
+          </span>
+        );
+      }
+    };
```

Comment:
I would move this function to a module level or class-level function / method. As is, the function will be redeclared on every re-render. sometimes we need to generate functions based on runtime values, so that makes sense to do, but in this case you should be able to write a function like this:

```js
const cameraIcon =  editable => {
  if ( editable ) {
    return <img
      src='/static/images/camera.png'
      alt="Camera image for profile image"
    />;
  }
};
```

Then in the `render` method you would just pull `editable` out of `this.props` and call `cameraIcon(editable)` inline in the JSX.